2025-02-26 19:49:36,088 INFO Watching for file changes with StatReloader
2025-02-26 19:52:42,497 INFO "GET / HTTP/1.1" 200 963
2025-02-26 19:52:43,955 WARNING Not Found: /favicon.ico
2025-02-26 19:52:43,959 WARNING "GET /favicon.ico HTTP/1.1" 404 4050
2025-02-26 19:52:50,415 INFO "GET /?q=count HTTP/1.1" 200 784
2025-02-26 19:52:58,495 INFO "GET /?q=adv HTTP/1.1" 200 966
2025-02-26 19:52:59,850 INFO "GET /books/2/ HTTP/1.1" 200 1683
2025-02-26 19:53:02,609 INFO "GET / HTTP/1.1" 200 963
2025-02-26 19:54:01,336 WARNING Not Found: /books/1/update/
2025-02-26 19:54:01,337 WARNING "GET /books/1/update/ HTTP/1.1" 404 4062
2025-02-26 19:55:43,926 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\books\urls.py changed, reloading.
2025-02-26 19:55:52,396 INFO Watching for file changes with StatReloader
2025-02-26 19:57:06,833 INFO Watching for file changes with StatReloader
2025-02-26 19:57:25,293 INFO "GET / HTTP/1.1" 200 963
2025-02-26 19:57:30,383 WARNING Not Found: /books/1/update/
2025-02-26 19:57:30,387 WARNING "GET /books/1/update/ HTTP/1.1" 404 4519
2025-02-26 19:57:39,802 INFO "GET /admin/ HTTP/1.1" 200 8905
2025-02-26 19:57:39,938 INFO "GET /static/admin/css/base.css HTTP/1.1" 304 0
2025-02-26 19:57:39,985 INFO "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
2025-02-26 19:57:40,014 INFO "GET /static/admin/js/theme.js HTTP/1.1" 304 0
2025-02-26 19:57:40,443 INFO "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
2025-02-26 19:57:40,619 INFO "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
2025-02-26 19:57:40,687 INFO "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
2025-02-26 19:57:40,694 INFO "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
2025-02-26 19:57:40,730 INFO "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
2025-02-26 19:57:40,730 INFO "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
2025-02-26 19:57:40,732 INFO "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
2025-02-26 19:57:44,910 WARNING Not Found: /update
2025-02-26 19:57:44,911 WARNING "GET /update HTTP/1.1" 404 4321
2025-02-26 19:58:14,232 WARNING Not Found: /books/1/update/
2025-02-26 19:58:14,233 WARNING "GET /books/1/update/ HTTP/1.1" 404 4519
2025-02-26 20:01:25,409 INFO Watching for file changes with StatReloader
2025-02-26 20:01:53,348 INFO "GET / HTTP/1.1" 200 963
2025-02-26 20:02:35,114 INFO "GET /books/2/update/ HTTP/1.1" 200 1954
2025-02-26 20:02:46,700 INFO "POST /books/2/update/ HTTP/1.1" 302 0
2025-02-26 20:02:46,719 INFO "GET / HTTP/1.1" 200 963
2025-02-26 20:06:23,289 INFO Watching for file changes with StatReloader
2025-02-26 20:06:26,856 INFO "GET / HTTP/1.1" 200 1200
2025-02-26 20:06:37,735 INFO "GET /books/2/update/ HTTP/1.1" 200 1954
2025-02-26 20:06:43,634 INFO "POST /books/2/update/ HTTP/1.1" 302 0
2025-02-26 20:06:43,663 INFO "GET / HTTP/1.1" 200 1200
2025-02-26 20:06:46,877 INFO "GET /books/2/ HTTP/1.1" 200 1683
2025-02-26 20:06:49,240 INFO "GET / HTTP/1.1" 200 1200
2025-02-26 20:06:54,937 INFO "GET /add/ HTTP/1.1" 200 2063
2025-02-26 20:08:02,370 INFO "POST /add/ HTTP/1.1" 302 0
2025-02-26 20:08:02,449 INFO "GET / HTTP/1.1" 200 1676
2025-02-26 20:08:04,894 INFO "GET /books/6/update/ HTTP/1.1" 200 1670
2025-02-26 20:08:10,703 INFO "POST /books/6/update/ HTTP/1.1" 302 0
2025-02-26 20:08:10,738 INFO "GET / HTTP/1.1" 200 1675
2025-02-26 20:08:15,669 INFO "GET /books/6/ HTTP/1.1" 200 1396
2025-03-03 23:20:18,545 INFO Watching for file changes with StatReloader
2025-03-03 23:20:20,280 INFO "GET / HTTP/1.1" 200 1634
2025-03-03 23:20:34,327 INFO "GET /add/ HTTP/1.1" 200 2365
2025-03-03 23:21:47,262 INFO "POST /add/ HTTP/1.1" 302 0
2025-03-03 23:21:47,283 INFO "GET / HTTP/1.1" 200 2286
2025-03-03 23:21:47,479 INFO "GET /media/book_covers/cover.jpg HTTP/1.1" 200 144278
2025-03-05 17:46:20,840 INFO Watching for file changes with StatReloader
2025-03-05 17:46:23,184 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 17:46:23,339 INFO "GET /media/book_covers/cover.jpg HTTP/1.1" 304 0
2025-03-05 17:46:23,455 WARNING Not Found: /favicon.ico
2025-03-05 17:46:23,455 WARNING "GET /favicon.ico HTTP/1.1" 404 5413
2025-03-05 17:46:29,095 INFO "GET /books/2/ HTTP/1.1" 200 2012
2025-03-05 17:46:33,406 ERROR Internal Server Error: /buy/2/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 113, in buy_book
    send_order_email(request.user, book)
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 107, in send_order_email
    send_mail(subject, message, 'your_email@gmail.com', [user.email])
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\__init__.py", line 88, in send_mail
    return mail.send()
           ^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\message.py", line 301, in send
    return self.get_connection(fail_silently).send_messages([self])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\backends\smtp.py", line 128, in send_messages
    new_conn_created = self.open()
                       ^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\backends\smtp.py", line 95, in open
    self.connection.login(self.username, self.password)
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 750, in login
    raise last_exception
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials a640c23a62f3a-abf3f3bbfb3sm889632866b.77 - gsmtp')
2025-03-05 17:46:33,422 ERROR "POST /buy/2/ HTTP/1.1" 500 112579
2025-03-05 17:47:43,396 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\myproject\settings.py changed, reloading.
2025-03-05 17:47:58,753 INFO Watching for file changes with StatReloader
2025-03-05 17:48:03,075 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 17:48:06,692 INFO "GET /books/7/ HTTP/1.1" 200 2203
2025-03-05 17:48:11,108 ERROR Internal Server Error: /buy/7/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 113, in buy_book
    send_order_email(request.user, book)
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 107, in send_order_email
    send_mail(subject, message, 'your_email@gmail.com', [user.email])
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\__init__.py", line 88, in send_mail
    return mail.send()
           ^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\message.py", line 301, in send
    return self.get_connection(fail_silently).send_messages([self])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\backends\smtp.py", line 128, in send_messages
    new_conn_created = self.open()
                       ^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\mail\backends\smtp.py", line 95, in open
    self.connection.login(self.username, self.password)
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 750, in login
    raise last_exception
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "C:\Users\Elene\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials a640c23a62f3a-abf600c58c1sm692516066b.127 - gsmtp')
2025-03-05 17:48:11,108 ERROR "POST /buy/7/ HTTP/1.1" 500 112534
2025-03-05 18:05:41,338 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\myproject\settings.py changed, reloading.
2025-03-05 18:05:42,655 INFO Watching for file changes with StatReloader
2025-03-05 18:14:24,113 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\myproject\settings.py changed, reloading.
2025-03-05 18:14:37,616 INFO Watching for file changes with StatReloader
2025-03-05 18:14:39,050 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:14:43,704 INFO "GET /books/2/ HTTP/1.1" 200 2012
2025-03-05 18:14:47,593 INFO "POST /buy/2/ HTTP/1.1" 302 0
2025-03-05 18:14:47,596 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:16:09,645 INFO "GET /admin/ HTTP/1.1" 200 8905
2025-03-05 18:16:09,705 INFO "GET /static/admin/css/base.css HTTP/1.1" 304 0
2025-03-05 18:16:09,705 INFO "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
2025-03-05 18:16:09,707 INFO "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
2025-03-05 18:16:09,713 INFO "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
2025-03-05 18:16:09,714 INFO "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
2025-03-05 18:16:09,715 INFO "GET /static/admin/js/theme.js HTTP/1.1" 304 0
2025-03-05 18:16:09,717 INFO "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
2025-03-05 18:16:09,833 INFO "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
2025-03-05 18:16:09,843 INFO "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
2025-03-05 18:16:09,844 INFO "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
2025-03-05 18:16:19,880 INFO "GET /admin/login HTTP/1.1" 301 0
2025-03-05 18:16:19,896 INFO "GET /admin/login/ HTTP/1.1" 302 0
2025-03-05 18:16:19,909 INFO "GET /admin/ HTTP/1.1" 200 8905
2025-03-05 18:16:23,599 INFO "GET /admin/login HTTP/1.1" 301 0
2025-03-05 18:16:23,618 INFO "GET /admin/login/ HTTP/1.1" 302 0
2025-03-05 18:16:23,641 INFO "GET /admin/ HTTP/1.1" 200 8905
2025-03-05 18:16:29,195 WARNING Not Found: /admin/register
2025-03-05 18:16:29,196 WARNING "GET /admin/register HTTP/1.1" 404 5928
2025-03-05 18:16:49,805 INFO "GET /login/ HTTP/1.1" 200 578
2025-03-05 18:17:50,335 WARNING Not Found: /change-password/
2025-03-05 18:17:50,336 WARNING "GET /change-password/ HTTP/1.1" 404 5428
2025-03-05 18:23:05,753 INFO "GET /change_password/ HTTP/1.1" 200 1383
2025-03-05 18:23:30,351 INFO "POST /change_password/ HTTP/1.1" 200 1629
2025-03-05 18:25:23,224 INFO Watching for file changes with StatReloader
2025-03-05 18:25:24,793 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:25:33,510 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 18:25:37,197 INFO "GET /reset_password/ HTTP/1.1" 200 405
2025-03-05 18:25:41,019 ERROR Internal Server Error: /reset_password/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 133, in reset_password
    form.save(request=request, from_email='your_email@gmail.com')
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\forms.py", line 494, in save
    self.send_mail(
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\forms.py", line 423, in send_mail
    body = loader.render_to_string(email_template_name, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 55, in render
    output = self.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader_tags.py", line 56, in render
    result = self.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'password_reset_confirm' not found. 'password_reset_confirm' is not a valid view function or pattern name.
2025-03-05 18:25:41,026 ERROR "POST /reset_password/ HTTP/1.1" 500 156915
2025-03-05 18:30:49,807 ERROR Internal Server Error: /reset_password/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 133, in reset_password
    form.save(request=request, from_email='your_email@gmail.com')
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\forms.py", line 494, in save
    self.send_mail(
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\contrib\auth\forms.py", line 423, in send_mail
    body = loader.render_to_string(email_template_name, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 55, in render
    output = self.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader_tags.py", line 56, in render
    result = self.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'password_reset_confirm' not found. 'password_reset_confirm' is not a valid view function or pattern name.
2025-03-05 18:30:49,807 ERROR "POST /reset_password/ HTTP/1.1" 500 156915
2025-03-05 18:31:41,847 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\urls.py changed, reloading.
2025-03-05 18:31:42,878 INFO Watching for file changes with StatReloader
2025-03-05 18:32:01,374 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\urls.py changed, reloading.
2025-03-05 18:32:01,987 INFO Watching for file changes with StatReloader
2025-03-05 18:35:16,103 INFO Watching for file changes with StatReloader
2025-03-05 18:35:17,587 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:35:22,820 ERROR Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 65, in login_
    return render(request, 'books/login.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'reset_password' not found. 'reset_password' is not a valid view function or pattern name.
2025-03-05 18:35:22,820 ERROR "GET /login/ HTTP/1.1" 500 115845
2025-03-05 18:35:23,142 ERROR Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 65, in login_
    return render(request, 'books/login.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'reset_password' not found. 'reset_password' is not a valid view function or pattern name.
2025-03-05 18:35:23,144 ERROR "GET /login/ HTTP/1.1" 500 115571
2025-03-05 18:38:49,769 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py changed, reloading.
2025-03-05 18:38:50,621 INFO Watching for file changes with StatReloader
2025-03-05 18:39:12,226 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py changed, reloading.
2025-03-05 18:39:13,142 INFO Watching for file changes with StatReloader
2025-03-05 18:39:50,239 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\urls.py changed, reloading.
2025-03-05 18:39:50,897 INFO Watching for file changes with StatReloader
2025-03-05 18:41:09,607 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\myproject\settings.py changed, reloading.
2025-03-05 18:41:10,195 INFO Watching for file changes with StatReloader
2025-03-05 18:41:50,545 INFO Watching for file changes with StatReloader
2025-03-05 18:41:52,045 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:41:54,435 INFO "GET /books/6/ HTTP/1.1" 200 1725
2025-03-05 18:41:57,610 INFO "POST /buy/6/ HTTP/1.1" 302 0
2025-03-05 18:41:57,626 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:42:40,979 ERROR Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 66, in login_
    return render(request, 'books/login.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'reset_password' not found. 'reset_password' is not a valid view function or pattern name.
2025-03-05 18:42:40,997 ERROR "GET /login/ HTTP/1.1" 500 115853
2025-03-05 18:42:41,443 ERROR Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py", line 66, in login_
    return render(request, 'books/login.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Elene\PycharmProjects\pythonProject\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'reset_password' not found. 'reset_password' is not a valid view function or pattern name.
2025-03-05 18:42:41,443 ERROR "GET /login/ HTTP/1.1" 500 115579
2025-03-05 18:46:07,764 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\urls.py changed, reloading.
2025-03-05 18:46:08,691 INFO Watching for file changes with StatReloader
2025-03-05 18:46:22,390 INFO Watching for file changes with StatReloader
2025-03-05 18:46:24,296 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:46:29,768 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 18:46:35,339 INFO "GET /reset_password/ HTTP/1.1" 200 3417
2025-03-05 18:46:35,616 INFO "GET /static/admin/css/forms.css HTTP/1.1" 200 8794
2025-03-05 18:46:35,637 INFO "GET /static/admin/css/widgets.css HTTP/1.1" 200 11564
2025-03-05 18:46:43,347 INFO "POST /reset_password/ HTTP/1.1" 302 0
2025-03-05 18:46:43,387 INFO "GET /password_reset/done/ HTTP/1.1" 200 2987
2025-03-05 18:47:01,704 INFO "GET /reset/MQ/cm7g5t-b326c7c463b52ac21e514f60aca297ca/ HTTP/1.1" 302 0
2025-03-05 18:47:01,784 INFO "GET /reset/MQ/set-password/ HTTP/1.1" 200 3874
2025-03-05 18:47:18,783 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 302 0
2025-03-05 18:47:18,812 INFO "GET /reset/done/ HTTP/1.1" 200 2825
2025-03-05 18:47:21,588 WARNING Not Found: /accounts/login/
2025-03-05 18:47:21,589 WARNING "GET /accounts/login/ HTTP/1.1" 404 6281
2025-03-05 18:48:51,249 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 18:49:49,021 INFO Watching for file changes with StatReloader
2025-03-05 18:49:50,616 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:49:53,974 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 18:49:58,438 INFO "GET /reset_password/ HTTP/1.1" 200 3417
2025-03-05 18:50:02,771 INFO "POST /reset_password/ HTTP/1.1" 302 0
2025-03-05 18:50:02,793 INFO "GET /password_reset/done/ HTTP/1.1" 200 2987
2025-03-05 18:50:11,532 INFO "GET /reset/MQ/cm7gbd-6fc068ebd2040318bb979689131ead0d/ HTTP/1.1" 302 0
2025-03-05 18:50:11,590 INFO "GET /reset/MQ/set-password/ HTTP/1.1" 200 3874
2025-03-05 18:50:22,391 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 200 3969
2025-03-05 18:50:34,224 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 200 3969
2025-03-05 18:50:53,572 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 302 0
2025-03-05 18:50:53,593 INFO "GET /reset/done/ HTTP/1.1" 200 2825
2025-03-05 18:50:55,266 WARNING Not Found: /accounts/login/
2025-03-05 18:50:55,266 WARNING "GET /accounts/login/ HTTP/1.1" 404 6281
2025-03-05 18:53:37,778 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\urls.py changed, reloading.
2025-03-05 18:53:38,665 INFO Watching for file changes with StatReloader
2025-03-05 18:54:55,182 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py changed, reloading.
2025-03-05 18:54:56,257 INFO Watching for file changes with StatReloader
2025-03-05 18:55:23,852 INFO C:\Users\Elene\PycharmProjects\pythonProject\myproject\myproject\books\views.py changed, reloading.
2025-03-05 18:55:24,442 INFO Watching for file changes with StatReloader
2025-03-05 18:56:44,163 INFO Watching for file changes with StatReloader
2025-03-05 18:56:45,756 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 18:56:48,829 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 18:56:50,570 INFO "GET /reset_password/ HTTP/1.1" 200 3417
2025-03-05 18:56:55,061 INFO "POST /reset_password/ HTTP/1.1" 302 0
2025-03-05 18:56:55,081 INFO "GET /password_reset/done/ HTTP/1.1" 200 2987
2025-03-05 18:57:02,331 INFO "GET /reset/MQ/cm7gmt-fea97f1184409ecd5ec96e10f52be616/ HTTP/1.1" 302 0
2025-03-05 18:57:02,376 INFO "GET /reset/MQ/set-password/ HTTP/1.1" 200 3874
2025-03-05 18:57:11,289 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 302 0
2025-03-05 18:57:11,308 INFO "GET /reset/done/ HTTP/1.1" 200 2825
2025-03-05 18:57:13,321 INFO "GET /accounts/login/ HTTP/1.1" 200 638
2025-03-05 18:57:20,285 INFO "POST /accounts/login/ HTTP/1.1" 200 697
2025-03-05 18:57:29,202 INFO "POST /accounts/login/ HTTP/1.1" 200 697
2025-03-05 18:57:30,982 INFO "GET /register/ HTTP/1.1" 200 1654
2025-03-05 18:58:46,851 INFO "GET /register HTTP/1.1" 301 0
2025-03-05 18:58:46,877 INFO "GET /register/ HTTP/1.1" 200 1654
2025-03-05 18:58:49,623 INFO "GET /reset_password/ HTTP/1.1" 200 3417
2025-03-05 18:58:57,307 INFO "GET /change_password/ HTTP/1.1" 200 1383
2025-03-05 18:59:14,970 INFO "POST /change_password/ HTTP/1.1" 200 1507
2025-03-05 18:59:30,154 INFO "GET /register/ HTTP/1.1" 200 1654
2025-03-05 18:59:47,466 INFO "POST /register/ HTTP/1.1" 200 1768
2025-03-05 19:00:00,348 INFO "POST /register/ HTTP/1.1" 302 0
2025-03-05 19:00:00,372 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 19:00:07,288 INFO "GET /login/ HTTP/1.1" 200 638
2025-03-05 19:00:19,524 INFO "POST /login/ HTTP/1.1" 302 0
2025-03-05 19:00:19,548 INFO "GET / HTTP/1.1" 200 2286
2025-03-05 19:00:26,554 INFO "GET /reset_password/ HTTP/1.1" 200 3417
2025-03-05 19:00:35,530 INFO "POST /reset_password/ HTTP/1.1" 302 0
2025-03-05 19:00:35,534 INFO "GET /password_reset/done/ HTTP/1.1" 200 2987
2025-03-05 19:00:43,180 INFO "GET /reset/MQ/cm7gsx-479a60b607ed2ae9cff2293230f83bbe/ HTTP/1.1" 302 0
2025-03-05 19:00:43,204 INFO "GET /reset/MQ/set-password/ HTTP/1.1" 200 3874
2025-03-05 19:00:52,568 INFO "POST /reset/MQ/set-password/ HTTP/1.1" 302 0
2025-03-05 19:00:52,589 INFO "GET /reset/done/ HTTP/1.1" 200 2825
2025-03-05 19:00:53,757 INFO "GET /accounts/login/ HTTP/1.1" 200 638
2025-03-05 19:01:04,286 INFO "POST /accounts/login/ HTTP/1.1" 200 697
2025-03-05 19:01:17,250 INFO "POST /accounts/login/ HTTP/1.1" 200 697
2025-03-05 19:01:30,686 INFO "POST /accounts/login/ HTTP/1.1" 302 0
2025-03-05 19:01:30,702 INFO "GET / HTTP/1.1" 200 2286
